Library/Homebrew/test/Gemfile

```diff
@@ -1,9 +1,13 @@
 source "https://rubygems.org"
 
-gem "mocha", "~> 1.1"
-gem "minitest", "~> 5.3"
-gem "rake", "~> 10.3"
-gem "parallel_tests", "~> 2.9"
+gem "minitest"
```

Comment:
I'd keep this, since we aren't using the one from the standard library, but a more up-to-date one.